packages/cascara/src/atoms/Flex/Flex.js

```diff
@@ -39,12 +41,14 @@ const propTypes = {
 };
 
 const Flex = ({
+  as = 'div',
   children,
   className,
   column = false,
   fluid = true,
   space = 'between',
   vAlign = 'start',
+  ...rest
 }) => {
   const classList = cx(className, {
     _: true,
```

Comment:
Just a heads up... this underscore class can be included directly in the arguments passed to the binded cx function. I don't think we need to change this now, as there will be one more big cleanup where we can do this... but we can simplify these where the base class is not part of the conditional logic.
```suggestion
  const classList = cx('_', className, {
```

Reply:
Understood

Reply:
This is still not done. We have the root underscore class in the options object instead of in the class list as the first argument.